modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -91,3 +92,52 @@ The Red Hat OpenShift Logging Operator redeploys the Fluentd pods. If the pods d
 ----
 $ oc delete pod --selector logging-infra=fluentd
 ----
+
+.Example: Setting a secret that contains a username and password
+
+You can use a secret that contains a username and password to authenticate a secure connection to an external Elasticsearch instance.
+
+For example, if you cannot use mutual TLS (mTLS) keys because a third party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password.
+
+. Create a `Secret` YAML file similar to the following example that contains base64 encoded values for `username` and `password`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-test-secret
+data:
+  username: dGVzdHVzZXJuYW1lCg==
+  password: dGVzdHBhc3N3b3JkCg==
+----
+
+. Create the secret:
++
+[source,terminal]
+----
+$ oc create secret -n openshift-logging openshift-test-secret.yaml
+----
+
+
+. Specify the name of the secret in your `ClusterLogForwarder`:
```

Comment:
```suggestion
. Specify the name of the secret in the `ClusterLogForwarder` resource:
```
General OCP rule is to have a word like "resource" or "object" after

Reply:
Updated. Thanks.